database/sql/V037__add_order_surplus_fee.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX ON orders (class);
```

Comment:
```suggestion
```

The queries that could be affected by this are solvable_orders and limit_orders_with_outdated_fees but they are going to filter out most orders through the valid_to condition on which there already is an index (for this reason). When the valid_to index has already been used then an extra index on the class doesn't do much.

Reply:
https://github.com/cowprotocol/services/pull/714/commits/f0d0ae1d94355946884d5b71d7be01afde7c8770